src/libs/ReportUtils.ts

```diff
@@ -9828,21 +9831,22 @@ function canReportBeMentionedWithinPolicy(report: OnyxEntry<Report>, policyID: s
 function prepareOnboardingOnyxData(
     introSelected: OnyxEntry<IntroSelected>,
     engagementChoice: OnboardingPurpose,
-    onboardingMessage: ValueOf<typeof CONST.ONBOARDING_MESSAGES>,
+    onboardingMessage: OnboardingMessagesType,
     adminsChatReportID?: string,
     onboardingPolicyID?: string,
     userReportedIntegration?: OnboardingAccounting,
     wasInvited?: boolean,
     companySize?: OnboardingCompanySize,
 ) {
+    const {onboardingMessages} = getOnboardingMessages();
```

Comment:
NAB. Let's call this function only if we are going to use `onboardingMessages` i.e. inside the if conditions